--- origsrc/launchy-2.5/platforms/unix/platform_unix_util.cpp	2010-02-13 13:37:54.000000000 -0600
+++ src/launchy-2.5/platforms/unix/platform_unix_util.cpp	2011-10-15 22:33:33.170129500 -0500
@@ -87,7 +87,7 @@ QString UnixIconProvider::getDesktopIcon
 	const char *dirs[] = { "/usr/share/applications/",
 			       "/usr/local/share/applications/",
 			       "/usr/share/gdm/applications/",
-			       "/usr/share/applications/kde/",
+			       "/usr/share/applications/kde4/",
 			       "~/.local/share/applications/"};
 	for(int i = 0; i < 5; i++) {
 	    QString dir = dirs[i];
